app/src/main/java/com/github/se/polyfit/viewmodel/meal/MealViewModel.kt

```diff
@@ -84,4 +84,10 @@ class MealViewModel(
     }
     _isComplete.value = _meal.value?.isComplete() ?: false
   }
+
+  // TODO: This can be removed once we are properly using a new ViewModel for each Meal
```

Comment:
I think that we have completely different views on what the lifespan of a `MealViewModel` should be. The way I see it, we only have one, but it seems you view this as creating a new one for each meal creation. Let's talk tomorrow after the stand-up meeting to make sure we are on the same page about this.